libnestutil/numerics.h

```diff
@@ -125,5 +125,10 @@ double dround( double );
  */
 double dtruncate( double );
 
+/**
+ * Returns true if n is integer
```

Comment:
Could you expand the documentation to point out that this means "integer up to rounding error"?